integration-tests/docker-compose.yml

```diff
@@ -9,6 +9,8 @@ services:
       - specific-mongo
       - specific-rabbitmq
       - dummy-service
+      - elastic
```

Comment:
Same, use the full name of elasticsearch, to keep things consistent.
The same for the rest of the file.